### ISSUE TITLE
Control of newline (EOL) character

### ISSUE THREAD

**Issue by koliyo** [lunet-io/scriban#192]
I have a problem with inconsistent file generation for windows/linux. I want to get LF line endings on both Linux and Windows in my generated files.

Is there some flag in scriban to control this, or some more general flag in C# environment. Sorry, not so experienced with C# ecosystem yet!

**Comment by xoofx** [comment]
You can modify this via [`TemplateContext.NewLine`](https://github.com/lunet-io/scriban/blob/5718821436a31c0e212452921fbee3fb045aebca/src/Scriban/TemplateContext.cs#L169)

**Comment by koliyo** [comment]
Thanks!

I found some additional places to consider.

To make sure the template files have consistent line endings:  in `.gitattributes` I have added
```
*.sbn eol=lf
```

Also, I was using `System.IO.StringWriter` for additional transformations, and this has additional setting `StringWriter.NewLine`.

It would be a lot simpler if the global [Environment.NewLine](https://docs.microsoft.com/en-us/dotnet/api/system.environment.newline?view=netframework-4.8) was writable! :)
